README.md

```diff
@@ -30,16 +30,16 @@ With yarn:
 yarn add graphql
 ```
 
-or alternatively using npm:
+alternatively using npm:
 
 ```sh
 npm install --save graphql
 ```
 
 GraphQL.js provides two important capabilities: building a type schema, and
-serving queries against that type schema.
+serving queries against that type of schema.
```

Comment:
@pranshuchittora I'm not sure about some of the changes in this PR, especially this one (`type schema` is a single entity here).
So I think this PR needs to be proof-read by a native speaker.

Reply:
Sounds good, looking forward for further review.